css/home-page.styl

```diff
@@ -422,6 +432,9 @@ $flex-section
       margin: 2em 22vw
       align-items: stretch
 
+      @media (max-width: 768px)
+        margin: 2em
+
```

Comment:
The text was too skinny on mobile: 
<img width="279" alt="screen shot 2019-01-23 at 10 11 18 pm" src="https://user-images.githubusercontent.com/3202211/51653513-b9bd7200-1f61-11e9-98d7-fac3fe9a8df5.png">

The fix: 
<img width="276" alt="screen shot 2019-01-23 at 10 10 57 pm" src="https://user-images.githubusercontent.com/3202211/51653515-bde98f80-1f61-11e9-9a79-0e8c1e929c16.png">